doc/panes/historylog.rst

```diff
@@ -17,6 +17,7 @@ Navigating the :guilabel:`History` pane is very straightforward.
 Each Spyder session is marked by a date and timestamp, making it easy to remember when you executed a certain command.
 Statements can be selected and copied from the context menu or with the normal system shortcuts.
 Just like in the Editor, selecting a word or phrase displays all other occurrences, and full syntax highlighting is also supported.
+The last ≈1000 lines entered are stored in the pane, which is currently not customizable (but may be increased in the future).
```

Comment:
I don't see the purpose of mentioning that it may be increased in the future, or that it is not customizable. It makes it feel like it is "wrong" somehow. What do you think?

Reply:
I agree, I would favor not including both of those, but I added the note that it is "not customizable" due to @ccordoba12 's comments on the original issue this is addressing in response to a user asking about clearing it,

> We should mention the limit on that section and that it's not possible to change it.

I then included the "may be increased in the future" as a result of that to help mitigate negative user sentiment surrounding an arbitrarily low limitation that was explicitly stated to be hard-coded, and reflect that it might be modified in the future (as discussed on #252 ).

Ideally we could get Carlos' opinion as the original issue author, but given how busy he is and how relatively minor this is, probably best for us to just decide now. So if you still agree it should be changed, we can go with that:

```suggestion
The last ≈1000 lines entered are stored in the pane.
```

Reply:
I agree that for now we shouldn't mention this.